src/pages/downloads.tsx

```diff
@@ -96,7 +97,12 @@ const Download = () => {
         <Spacer h={3} />
         <Text className="header" h1>
           <TranslatableText>Give your PC an Upgrade.</TranslatableText>
-        </Text>
+              </Text>
+              <Spacer />
+              <Note style={{ display: 'inline-block' }} type="warning" label="NOTE">
+                  <TranslatableText link="/demo">The download is not available yet, but you can always [%a]check out[%a] the demo.</TranslatableText>
```

Comment:
~~The download is not available yet, but you can always [%a]check out[%a] the demo.~~
The download is not available yet, but you can always check out the [%a]demo[%a].